docs/user-guide/media-editor-window.md

```diff
@@ -0,0 +1,68 @@
+# Media Editor Window
+**********************
+
+The **Change media source** button allows to access the Media Editor:
+
+<img src="../img/media-editor-window/media-editor.jpg" class="ms-docimage"/>
+
+Through this window, you can add or edit three different types of media:
+
+* Images
+
+* Videos 
+
+* Maps
+
+## Images
+
+In order to add an image, the stroy editor can click on the **Images** tab <img src="../img/button/images.jpg" class="ms-docbutton"/> in order to switch to the images section. In this section of the media editor window, with a click on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/>, it is possible to define the image settings.
+
+<img src="../img/media-editor-window/img-settings.jpg" class="ms-docimage"/>
+
+In particular, it is possible to insert the following parameters:
+
+* The **Source** of the image (its URL)
+
+* The **Title** of the image
+
+* The **Alternative text**, that appears if the link is broken
+
+* A **Description**, used to explain the contents of the image. The description is available in the images preview list and, if the image is added as content in the *Pharagraph section*, in the *Immersive section* or in the *Media section*, under the image, as follows:
+
+<img src="../img/media-editor-window/image-description.jpg" class="ms-docimage" width="400px"/>
+
+* The **Credits**, displayed on the bottom-right of the image
+
+!!! warning
+    **Source** and **Title** are mandatory fields.
+
+With a click on **Save** <img src="../img/button/save-icon.jpg" class="ms-docbutton"/>, the image is included in the list of the available images ready to be selected for the current story. The image also becomes immediately available on preview.
+
+<img src="../img/media-editor-window/img-preview.jpg" class="ms-docimage"/>
+
+!!! note
+    It is always possible to change an image settings through the **Edit** button <img src="../img/button/editing-button.jpg" class="ms-docbutton"/> also in a second time.
+
+Once selectd in the list, the image can be included in the story by clicking on the **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.
+
+## Maps
+
+In order to add a map to the story, the story editor can click on the **Maps** tab <img src="../img/button/maps-button.jpg" class="ms-docbutton"/> in order to swich to the map section of the Media Editor. Here the list of maps ready to be used for the story is available: the editor can search and select a map in the list to apply it in the story or create a map from scratch by clicking on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/> to open the [Map Editor](configure-map.md#advanced-map-editor).
```

Comment:
To add a map to the story, the story editor can click on the **Maps** tab <img src="../img/button/maps-button.jpg" class="ms-docbutton"/> and so  swich to the map section of the Media Editor...